UefiTestingPkg/AuditTests/DMAProtectionAudit/UEFI/IVRS/DMAProtectionTestArch.c

```diff
@@ -47,6 +47,13 @@ CheckExcludedRegions (
   UINT32                 EfiDescriptorVersion;
   IVMDListNode           *Head;
   IVMDListNode           *Current;
+  UINTN                  MemoryTypesIndex;
+  BOOLEAN                IsAllowed;
+  UINTN                  AllowedMemoryTypesCount;
+
+  EFI_MEMORY_TYPE  AllowedMemoryTypes[] = { EfiReservedMemoryType, EfiACPIMemoryNVS };
+
+  AllowedMemoryTypesCount = sizeof (AllowedMemoryTypes) / sizeof (AllowedMemoryTypes[0]);

```

Comment:
Typically, `ARRAY_SIZE()` is used for this.

```suggestion
  AllowedMemoryTypesCount = ARRAY_SIZE (AllowedMemoryTypes);
```

Reply:
thanks, done.